src/benchmark_register.cc

```diff
@@ -215,6 +215,7 @@ Benchmark::Benchmark(const char* name)
   ComputeStatistics("mean", StatisticsMean);
   ComputeStatistics("median", StatisticsMedian);
   ComputeStatistics("stddev", StatisticsStdDev);
+  ComputeStatistics("cv", StatisticsRelStdDev, kPercentage);
```

Comment:
if we're going to call it `RelStdDev` in the code, maybe we should report it as "relstddev" too.

Reply:
Ok, let's be consistent :)